src/libPMacc/include/algorithms/math/doubleMath/sqrt.tpp

```diff
@@ -50,7 +50,7 @@ struct RSqrt<double>
 
     HDINLINE double operator( )(const double& value )
     {
-#if defined(_MSC_VER) && !defined(__CUDA_ARCH__)
```

Comment:
offline discussion with @BenjaminW3:
we needed `_MSC_VER` for `sincos` in `.../trigo.tpp`. Here it is only a copy and past remanent and not required.